packages/server/modules/previews/index.js

```diff
@@ -252,33 +235,33 @@ exports.init = (app) => {
         id: req.params.commitId
       })
       if (!commit) return res.sendFile(noPreviewImage)
-
+      console.log(req.params.angle || DEFAULT_ANGLE)
```

Comment:
Are all of the console.log calls meant to be there or is that leftover debugging?

Reply:
nope 😅